CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ add_executable(${EXE_NAME}
 		src/vk/descriptors/descriptorSetInfo.cpp
 		include/vk/cmdSync.h
 		src/vk/cmdSync.cpp
-		include/app/config/json.h
+		include/app/config/JSON.h
```

Comment:
Why apper case?

Reply:
That's our code style, see
https://github.com/MoonDweller29/vulkan_3d/wiki/Code-style#file-names
> (Upper/lower case is preserved.)